app/javascript/packages/document-capture/components/review-issues-step.tsx

```diff
@@ -55,14 +57,44 @@ function ReviewIssuesStep({
   isFailedDocType = false,
   pii,
   captureHints = false,
+  failedImageFingerprints = { front: [], back: [] },
 }: ReviewIssuesStepProps) {
   const { trackEvent } = useContext(AnalyticsContext);
   const [hasDismissed, setHasDismissed] = useState(remainingAttempts === Infinity);
   const { onPageTransition, changeStepCanComplete } = useContext(FormStepsContext);
+  const [skipWarning, setSkipWarning] = useState(false);
   useDidUpdateEffect(onPageTransition, [hasDismissed]);
 
-  const { onFailedSubmissionAttempt } = useContext(FailedCaptureAttemptsContext);
-  useEffect(() => onFailedSubmissionAttempt(), []);
+  const { onFailedSubmissionAttempt, failedSubmissionImageFingerprints } = useContext(
+    FailedCaptureAttemptsContext,
+  );
+  useEffect(() => onFailedSubmissionAttempt(failedImageFingerprints), []);
+
+  useLayoutEffect(() => {
+    let frontMetaData: any = null;
```

Comment:
Can we use a more useful type than `any` here, if we know it?

Reply:
I use a anoymous type here and thought it should be a type of ImageAnalyticsPayload, but not 100% sure.